cfme/tests/webui/test_general_ui.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
+from cfme.utils.appliance.implementations.ui import navigate_to
+
+
+@pytest.fixture()
+def import_tags(appliance):
+    client = appliance.ssh_client
+    client.run_command('cd /var/www/miq/vmdb/lib/tasks/')
```

Comment:
I think we need some assertions in this chain of commands and output processing, to ensure that nothing is going wrong and cleanup+skip the test if so.

Consider asserting against each of these commands, so their RC is checked (if you expect 0 RC), and wrapping the whole block of commands in a `try/except` in order to catch the `AssertionError` and delete files + `pytest.skip()` with an explanation.